pipelines/batch/src/main/java/com/google/fhir/analytics/FetchSearchPageFn.java

```diff
@@ -128,6 +129,7 @@ abstract class FetchSearchPageFn<T> extends DoFn<T, KV<String, Integer>> {
 
   protected AvroConversionUtil avroConversionUtil;
 
+  @SuppressWarnings("NullAway.Init")
```

Comment:
nit: consider adding a comment why we have this here.

Reply:
Done